### ISSUE TITLE
Strategy 3: Hyperparameter testing

### ISSUE THREAD

**Issue by ge96lip** [ge96lip/Explore-Transfer-Learning#5]
- Different learning rates and/or learning rate schedulers for different layers.
- Benefit of applying data augmentation during training (flip, small rotations, crops, small
size scaling) and also L2 regularization.
- Effect of fine-tuning or not the batch-norm parameters and updating the estimate of the
batch mean and standard deviations on the final performance on the new dataset
